app/javascript/src/components/Dashboard/Notes/Note/Content.jsx

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+
+import { Typography } from "neetoui";
+
+const Content = ({ text }) => (
```

Comment:
This needn't be a separate component. Refer prev comments.

Reply:
Removed this component.